.github/workflows/tf-test.yml

```diff
@@ -0,0 +1,64 @@
+name: "TF template tests"
+on: [workflow_dispatch, pull_request]
+
+jobs:
+  ValidateTF:
+    runs-on: ubuntu-latest
+    name: "Validate Terraform module"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: terraform validate
+        uses: dflook/terraform-validate@v1
+        with:
+          directory: 'aws-observability-terraform/'
+
+  ValidateLinting:
+    runs-on: ${{ matrix.os }}
+    name: "Terraform template linting verification"
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
do we have to run tflint for every os ?

Reply:
Yes, it's not required. Will use Ubuntu OS only.